src/libs/auth.js

```diff
@@ -119,7 +119,11 @@ export const signIn = async (includeBillingScope = false) => {
 
 export const reloadAuthToken = (includeBillingScope = false) => {
   const args = getSigninArgs(includeBillingScope)
-  return getAuthInstance().signinSilent(args).catch(() => false)
+  return getAuthInstance().signinSilent(args).catch((e) => {
+    console.error("An unexpected exception occurred while attempting to refresh auth credentials.")
+    console.error(e)
```

Comment:
do we have tooling setup to guzzle up console.error client-side logs like this and pump them into our monitoring mecha nics?

Reply:
@msilva-broad i have no clue, id assume you could find them in gcp but ill double check before merging this